src/style.css

```diff
@@ -1,3 +1,72 @@
+:root {
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+h1 {
+  justify-self: center;
```

Comment:
The `justify-self` property is used in grid layouts, but the `h1` element is not within a grid container. Consider using `text-align: center;` if you intend to center the text within its container.